doc/user-guides/acrn-sanity-test.rst

```diff
@@ -0,0 +1,342 @@
+#. Check CPU status by “cat /proc/cpuinfo”
+
+* Expected Result
+
+1. Step 2: Expect to see all CPUs under SOS
+#. Step 4: Expect to see 1 core under UOS since launch_uos.sh pass-through 1 core
+
+ST_HV_Memory_LaaG_Default_Partition_Check
+*****************************************
+
+* Description
+
+This test case focus on checking if the pass through memory to Guest OS and 
+check if the memory size is in a reasonable scope as launch_uos.sh defined. 
+Since the Guest OS will take a part of memory, the available memory size will 
+not be exact as what pass-through.
+
+* Prerequisites
+
+Service OS and Guest OS boot up normally
+
+* Test Steps
+
+1. Boot to SOS
+#. Boot LaaG UOS with default memory setting in launch_uos.sh
+#. Check ‘cat /proc/meminfo’ and see if the available memory is close the size passed through to UOS
+
+Expected Result
+
+1. Boot to UOS successfully
+#. Step 3: UOS - Depends on parameter passed to acrn-dm in launch_uos.sh.
+
+ST_HV_DM_LaaG_DefaultCPU_2GMemory_Conf
+**************************************
+
+* Description
+
+This test case focus on check if the pass through memory and CPU with certain 
+value in launch_uos.sh are correct.
+
+* Prerequisites
+
+Service OS and Guest OS boot up normally
+
+* Test Steps
+
+1. Boot to SOS
+#. Create UOS with 1core and memory=2048
+#. Check “cat cat /proc/cpuinfo” and “cat /proc/meminfo”
+
+* Expected Result
+
+1. Boot to UOS successfully
+#. Step 3: UOS memory size and CPU cores depends on parameter passed to acrn-dm in launch_uos.sh.
+
+IO
+**
+
+ST_ACRNIO_LaaG_Virtio_net
+*************************
+
+* Description
+
+This test case focus on checking if Service OS and LaaG can be allocated different IP, 
+and ping is ok by sharing same physical network card.
+
+* Prerequisites
+
+Need support Virtio, and bridge/tap is created normally in Service OS
+
+* Test Steps
+
+1. Boot DUT, and enter SOS from acrn hypervisor
+
+   .. code-block:: none
+
+      $ vm_console
+
+#. Check IP and bridge/tap after log in sos
+
+   .. code-block:: none
+     
+      $ ip a
+      $ brctl show
+      bridge name     bridge id               STP enabled     interfaces
+      acrn-br0         8000.1ad4ac453b20       no              acrn_tap0
+                                                               ens4
+
+#. Boot UOS using the launch_uos.sh 
+
+   .. code-block:: none
+   
+      $./launch_uos.sh
+
+#. Check UOS IP address
+
+* Expected Result
+
+IP address should be allocated for both SOS and UOS, and ping is ok
+
+ST_ACRNIO_LaaG_Virtio_storage
+*****************************
+
+* Description
+
+This test case focus on checking if LaaG using virtio storage can boot up successfully.
+
+* Prerequisites
+
+Need support Virtio
+
+* Test Steps
+
+1. In SOS: copy or create one img file as rootfs.
+#. Boot UOS using this img file in cmd line.
+
+   .. code-block:: none
+
+   -s 4,virtio-blk,/data/android/android.img \ (for android)
```

Comment:
do we have a guide for how to create/get the image file? 
should remove android.img which is not open source.